apps/reference/_supabase_dart/generated/storage-from-upload.mdx

```diff
@@ -80,8 +78,8 @@ The body of the file to be stored in the bucket.
   <div class="method-list-item-description">
 
 HTTP headers.
-`cacheControl`: string, the `Cache-Control: max-age=<seconds>` seconds value.
-`contentType`: string, the `Content-Type` header value. Should be specified if using a `fileBody` that is neither `Blob` nor `File` nor `FormData`, otherwise will default to `text/plain;charset=UTF-8`.
+`cacheControl`: string, the `Cache-Control: max-age=<seconds>` seconds value. <br />
+`contentType`: string, the `Content-Type` header value. The SDK will read the content type and set the header for you by default.<br />
```

Comment:
The formatting here is kind of hard to parse:
![Screen Shot 2022-09-08 at 12 23 26 PM](https://user-images.githubusercontent.com/7026076/189208583-04208894-859c-4d70-8fbd-7fd7e8060d31.png)

Can we make this a "properties" table (e.g., [signUp()](https://supabase.com/docs/reference/javascript/auth-signup#parameters))?
![Screen Shot 2022-09-08 at 12 24 44 PM](https://user-images.githubusercontent.com/7026076/189208697-5470b056-be91-4cc0-b38a-ee48b501f64d.png)

Reply:
Sounds good! Let me do this now!

Reply:
Done!